### PR TITLE
remove "..." from strings.

### PR DESCRIPTION
**Pull request by YtxCash**
I am working on translation from English to Chinese, Weblate.

There are lots of "..." in menu and somewhere, I do not know why contributors add them.

In my feeling, they are not so necessary. 

After remove them, the software looks more clear, so I remove almost all "..." from strings. Only in strings.

### CODE REVIEW

**Comment by christopherlam** [comment]
It would seem that HIGs tend to support your view: ellipses only to be used if "further input is necessary":

https://developer.gnome.org/hig/stable/writing-style.html.en#ellipses
https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-anatomy/
https://docs.microsoft.com/en-us/windows/win32/uxguide/ctrl-command-buttons (Design Concepts)

However, (my opinion only) this UI change is jarring, and will bring out a lot of opiniated views... (such as mine!)

**Comment by fellen** [comment]
https://bugs.gnucash.org/show_bug.cgi?id=792975
Imagine a menu entry
1. `Delete files` will immediately remove them, but
2. `Delete files…` or
3. `Open Cleanup Tool` would open a dialog to specify the selection and/or ask for confirmation before execution.

Tasks:
1. Unification: we should replace 3 dots by one typographical horizontal ellipse. 
2. Verification: Bug Comment 12 lists a few places, where we should remove them..

Each should ideally be done in the sources and the po files in one PR.

File: gnucash/gnome-utils/assistant-xml-encoding.c [hunk 1]
```diff
@@ -1091,7 +1091,7 @@ gxi_parse_file (GncXmlImportData *data)
     }
 
     xaccLogDisable ();
-    gxi_update_progress_bar (_("Reading file..."), 0.0);
```
Comment (fellen) [inline gnucash/gnome-utils/assistant-xml-encoding.c:1094]: No, this are status messages. Here the Ellipses means:
Work in progress, be patient!

**Comment by TianXing-Yi** [comment]
> https://developer.gnome.org/hig/stable/writing-style.html.en#ellipses
> https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-anatomy/
> https://docs.microsoft.com/en-us/windows/win32/uxguide/ctrl-command-buttons

> Proper use of ellipses is important to indicate that users can make further choices before performing the action, or even cancel the action entirely. The visual cue offered by an ellipsis allows users to explore your software without fear.

Documents are helpful.

But ellipses are less useful for user. Like me, use Gnucash since 2014, for company, for home. Until today I know "..." means opening a dialog.  In other words, it doesn't matter if I don't know.

In my option, ellipses are more usefull for developers. For user, ellipses mean warning will be better, like Print... Delete...

There operation will not directly change user data should not use "...", like Import **, Export **, Find, Filter.
Otherwise, software will make user feel a bit verbose. It's my feeling.

File: gnucash/gnome-utils/assistant-xml-encoding.c (hunk 1 above)
Comment (TianXing-Yi) [reply gnucash/gnome-utils/assistant-xml-encoding.c:1094]: > No, this are status messages. Here the Ellipses means:
> Work in progress, be patient!
> [Edit:]
> Often further details or a progress bar follow.

Yeah , it's suitable here. I made a mistake.

**Comment by TianXing-Yi** [comment]
@fellen Hi sir, Is there a way that remove "..." only from Chinese? 
Weblate checks match.  The changes cannot through it.

**Comment by fellen** [comment]
> @fellen Hi sir, Is there a way that remove "..." only from Chinese?
> Weblate checks match. The changes cannot through it.

The docs about checks start with https://docs.weblate.org/en/latest/user/checks.html. I did not find a way.

I am a big fan of international standards. They make many things easier. Sometimes rulesets get improved by others like from the File Hierarchy Standard (FHS) to the XDG standards. But as a inrtnational comunity we should try to stay close to the standards

I think, we have a common target:
Remove the dots from the menu entries listed in https://bugs.gnucash.org/show_bug.cgi?id=792975#c12 and perhaps some which Ii did not see on the first glimpse. That is the point where you should start.

**Comment by jralls** [comment]
@YtxCash We'll want to merge this in December as preparation for the 4.900 release in January. Can you rebase it on a current master and do whatever you need to in order to make it ready for review?

**Comment by Bob-IT** [comment]
@jralls Can this be closed, looks like your changes replace these.

**Comment by jralls** [comment]
> looks like your changes replace these.

Indeed they do, I just forgot to close this out.

**Pull request closed without merging**